src/Components/LogExplorationPage.tsx

```diff
@@ -1,11 +1,16 @@
 import React, { useEffect, useState } from 'react';
 
-import { getUrlSyncManager } from '@grafana/scenes';
-import { newLogsExploration } from 'services/scenes';
+import { SceneTimeRange, getUrlSyncManager } from '@grafana/scenes';
 import { IndexScene } from './IndexScene/IndexScene';
+const DEFAULT_TIME_RANGE = { from: 'now-15m', to: 'now' };
 
 export const LogExplorationPage = () => {
-  const [exploration] = useState(newLogsExploration());
+  // Here we are initializing the scene with the default time range
+  const [exploration] = useState(
```

Comment:
I think the `useState` can also be removed?

Reply:
I am not sure cause `LogExplorationPage` >might? re-render which could potentially create new index scene if it wouldn't be memoized/useRef. We can update it in the future if needed - I am not sure why there are 2 component either in here. But let's make "future us" problem. :D